### DISCUSSION TITLE
Is anyone working on integrating something like Continue or towards supporting local Ollama models?

### DISCUSSION THREAD

**Discussion by dhruvkelawala** [zed-industries/zed#12241] [Feature Requests]
Hello, I wanted to ask if someone was already working on integrating https://github.com/continuedev/continue to support local Ollama models? The community is asking for it and if no ones working on it, I would love to help build it!

**Comment by dhruvkelawala** [comment]
Raising this again

**Comment by Moshyfawn** [comment]
Be sure to follow this thread
- #4424
